R/ospsuite.utils-env.R

```diff
@@ -0,0 +1,45 @@
+# Environment that holds various global variables and settings for the ospsuite.utils,
+# It is not exported and should not be directly manipulated by other packages.
+ospsuiteUtilsEnv <- new.env(parent = emptyenv())
+
+# name of the package. This will be used to retrieve information on the package at run time
+ospsuiteUtilsEnv$packageName <- "ospsuite.utils"
+
+ospsuiteUtilsEnv$suiteName <- "Open Systems Pharmacology"
+
+# Separator defined in OSPSuite.Core.
+ospsuiteUtilsEnv$pathSeparator <- "|"
```

Comment:
Is this necessary for the current package?

Reply:
Not for the current, but this is a property that can be used by all packages working with OSPS models - ospsuite-r, parameter identification, ReportingEngine, esqlabsr... It's like the "system path separator".

Reply:
Ok but this can be read from ospsuite-r right?

Reply:
Since `ospsuiteUtilsEnv` is not exported in the namespace here, I think you would need to read it using `ospsuite.utils:::ospsuiteUtilsEnv$pathSeparator`

I have seen in other packages that similar settings were exported with the prefix `.` (e.g. `ggplot2::.pt`).
I was wondering if anyone knows why. And if we should do the same here (`.pathSeparator`) ?

Reply:
Maybe this is a little bit of a leak that this is used throughout when. We have high level methods to convert string to path and path to string

Reply:
Moved back to ospsuite.